uelc/templates/pagetree/page.html

```diff
@@ -51,12 +51,23 @@
 {% endfor %}
 
 {% if block.block.display_name == "Gate Block" %}
-<div class="pageblock{% if block.css_extra %} {{block.css_extra}}{% endif %}">
-    {% render block %}
-    <p class="unlock-msg alert alert-success" style="display:none">
-        You may proceed to the next section
+
+
+  {% if not gate_submission %}
+    <p class="wait-msg alert alert-danger hidden curveball-alert-danger">
+        Please wait for the facilitator to allow you to proceed.
     </p>
-</div>
+  {% endif %}
+
+  {% if section_submission and not gate_submission%}
```

Comment:
There are supposed to be 2 messages. One on top of the page and one on the bottom.